app/presenters/spent_tag.rb

```diff
@@ -0,0 +1,38 @@
+class SpentTag
+  GREEN = 'green'.freeze
+  RED = 'red'.freeze
+
+  COLORS = {
+    ResultsVariant::SPENT => GREEN,
+    ResultsVariant::NOT_SPENT => RED,
+    ResultsVariant::NEVER_SPENT => RED,
+    ResultsVariant::SPENT_SIMPLE => GREEN,
+    ResultsVariant::INDEFINITE => RED,
+  }.freeze
+
+  attr_reader :spent_date
+
+  def initialize(spent_date:)
+    @spent_date = spent_date
+  end
+
+  def color
+    if spent_date.instance_of?(Date)
```

Comment:
Is it possible here you just do `COLORS[variant]` as `#variant` method already do their magic?

Reply:
Good shout! 👍